### ISSUE TITLE
AVT 1 - Skeleton: Breadcrumb has a DAP violation for missing hyperlink label

### ISSUE THREAD

**Issue by snidersd** [carbon-design-system/carbon#4178]
##Environment
macOS Mojave version 10.14.6
Chrome Version Version 77.0.3865.90
Carbon v10 - React
DAP IBM Accessibility 1.8.0.0 Sept 2019 Ruleset

##Detailed Description
Go to the React skeleton for Breadcrumb
Run DAP and the following issue is found on the on each skeleton component (see screenshots):
<img width="1323" alt="breadcrumb" src="https://user-images.githubusercontent.com/21676914/66067641-a1025480-e511-11e9-8120-099f4345b7f8.png">
